docs/developers/dev-guide.md

```diff
@@ -60,19 +60,44 @@ L1 transactions can be divided in two groups depending the originator of the tra
 - **L1 Coordinator Transactions**: originate from the coordinator.
 
 #### L1 User Transactions
-L1 user transactions are received by the smart contract. These transactions are concatenated and added in queues to [`force the coordinator`](../developers/dev-guide?id=L1L2-Batches) to process them as part of the batch. The queue that will be forged in the next L1L2-batch it's always frozen, and the L1 Transactions will be added in the following queues. In case a transaction is invalid (e.g. try to send more amount than the account has) will be processed by the circuit but will be nullified.
+L1 user transactions (L1UserTxs) are received by the smart contract. These transactions are concatenated and added in queues to [`force the coordinator`](../developers/dev-guide?id=L1L2-Batches) to process them as part of the batch. The queue that will be forged in the next L1L2-batch it's always frozen, and the L1 Transactions will be added in the following queues. In case a transaction is invalid (e.g. try to send more amount than the account has) will be processed by the circuit but will be nullified.
```

Comment:
```suggestion
L1 user transactions (L1UserTxs) are received by the smart contract. These transactions are concatenated and added in queues to [`force the coordinator`](../developers/dev-guide?id=L1L2-Batches) to process them as part of the batch. The queue that will be forged in the next L1L2-batch is always frozen, and the L1 Transactions will be added in the following queues. In case a transaction is invalid (e.g. attempts to send an amount greater than the account balance) it will be processed by the circuit but will be nullified.
```